testsuite/features/step_definitions/api_common.rb

```diff
@@ -630,3 +630,32 @@
 When(/^I delete profile and distribution using the API for "([^"]*)" kickstart tree$/) do |distro_name|
   $api_test.kickstart.tree.delete_tree_and_profiles(distro_name)
 end
+
+When(/I verify channel "([^"]*)" is( not)? modular via the API/) do |channel_label, not_modular|
```

Comment:
is it nitpicky to ask for `is (not)?` or did it have to be this way due to some constraint?

Reply:
It gets a bit tricky with whitespaces.

`is (not)? modular` would expect two of them between `is` and `modular` when we do not write the `not`

I don't like it either, but I did not want to add other step definitions just to negate the results